core/src/main/java/org/keycloak/representations/idm/RealmRepresentation.java

```diff
@@ -857,6 +859,27 @@ public void setClientAuthenticationFlow(String clientAuthenticationFlow) {
         this.clientAuthenticationFlow = clientAuthenticationFlow;
     }
 
+    public String getDockerAuthenticationFlow() {
+        return dockerAuthenticationFlow;
+    }
+
+    public RealmRepresentation setDockerAuthenticationFlow(final String dockerAuthenticationFlow) {
+        this.dockerAuthenticationFlow = dockerAuthenticationFlow;
+        return this;
+    }
+
+    public List<String> getEnabledFeatures() {
```

Comment:
Please remove. This information is available on server info (serverInfo.profileInfo.)